process/css-styleguide/spacing.css

```diff
@@ -0,0 +1,349 @@
+@import "./default.css";
+
+.p-0 {
+  padding: var(--size-0);
+}
```

Comment:
I know this is not a component, but I think this should also have a `.html` file with all the cases + a section in the readme, because this is a very useful utility :)